primitive-types/impls/serde/benches/impl_serde.rs

```diff
@@ -12,46 +12,36 @@
 //! cargo bench
 //! ```
 
-#[macro_use]
-extern crate criterion;
-#[macro_use]
-extern crate uint;
-#[macro_use]
-extern crate impl_serde;
-extern crate serde_json;
+use criterion::*;
+use impl_serde::*;
+use serde_json::*;
+use uint::*;
```

Comment:
Please try to restrict imports to what is actually needed (helps future readers to understand at a glance what the code below does).

```suggestion
use uint::construct_uint;
```

Reply:
In this case it might be ok to import *, but in others we'd like to see what's being imported. See https://github.com/paritytech/parity-common/pull/205/files